toolchain/check/node_stack.h

```diff
@@ -405,7 +405,7 @@ class NodeStack {
       case Parse::NodeKind::VariableInitializer:
       case Parse::NodeKind::VariableIntroducer:
         return IdKind::SoloParseNode;
-// Use x-macros to handle token cases.
+// Use x-macros to handle operator and token cases.
```

Comment:
```suggestion
// Use x-macros to handle boilerplate cases.
```

Maybe a more broad statement? (I hadn't changed this before because of operators being tokens too -- a different interpretation of the word)